File: add/codesnippet/CPP/e-system.windows.forms.c_6_1.cpp
   private:
      void AddContextMenuChangedHandler()
      {
         this->myTextBox->ContextMenuChanged += gcnew EventHandler( this, &MyForm::TextBox_ContextMenuChanged );
      }

      void TextBox_ContextMenuChanged( Object^ /*sender*/, EventArgs^ /*e*/ )
      {
         MessageBox::Show( L"Shortcut menu of TextBox is changed." );
      }

File: add/codesnippet/CSharp/e-system.windows.forms.c_6_1.cs
      private void AddContextMenuChangedHandler()
      {
         this.myTextBox.ContextMenuChanged += new EventHandler(TextBox_ContextMenuChanged);
      }

      private void TextBox_ContextMenuChanged(object sender, EventArgs e)
      {
         MessageBox.Show("Shortcut menu of TextBox is changed.");
      }